### PR TITLE
Let reindexing resume from FAILED state after 10 minutes 

### PR DESCRIPTION
**Pull request by jonmv**
@bjorncs please review.

### CODE REVIEW

**Comment by jonmv** [comment]
@vekterli FYI. I haven't seen any other failures than `NO_SPACE`. I can't imagine what errors should correctly stop this, permanently. Can you think of any?

**Comment by vekterli** [comment]
@jonmv nothing that I can think of off the top of my head 🦄

**Review by bjorncs — APPROVED** [review]

File: clustercontroller-reindexer/src/main/java/ai/vespa/reindexing/Reindexer.java [hunk 1]
```diff
@@ -133,14 +135,15 @@ static Reindexing updateWithReady(Map<DocumentType, Instant> ready, Reindexing r
...
-                log.log(WARNING, "Unknown reindexing state '" + status.get().state() + "'");
-            case FAILED:
-                log.log(FINE, () -> "Not continuing reindexing of " + type + " due to previous failure");
+                log.log(WARNING, "Unknown reindexing state '" + status.get().state() + "'—not continuing reindexing of " + type);
             case SUCCESSFUL: // Intentional fallthrough — all three are done states.
                 return;
             case RUNNING:
                 log.log(WARNING, "Unexpected state 'RUNNING' of reindexing of " + type);
                 break;
+            case FAILED:
```
[the remark this answers is not in the document]
Comment (jonmv) [reply clustercontroller-reindexer/src/main/java/ai/vespa/reindexing/Reindexer.java:144]: Hmm ... I like it better then way it is, but thanks :)

**Comment by jonmv** [comment]
Oh master of merges, guardian of git, please accept my plea for merging to master this humble commit.

**Pull request merged**